seml/observers.py

```diff
@@ -162,6 +168,10 @@ def __init__(
             ":white_check_mark: *{experiment[name]}* "
             "completed after _{elapsed_time}_ with result=`{result}`"
         )
+        self.started_text = started_text or (
+            ":hourglass_flowing_sand: *{experiment[name]}* "
+            "started on host `{host_info[hostname]}` at _{start_time}_."
+        )
```

Comment:
Should we include the default texts as default parameters in the function?

Reply:
That's the normal way and more discoverable by the user.

Reply:
Done in 6f6c5c2.

Reply:
I meant as actual default parameters, but this works as well.